test/e2e/main_test.go

```diff
@@ -142,3 +143,29 @@ func TestMemoryUsageRecordingRule(t *testing.T) {
 		"count(namespace:container_memory_usage_bytes:sum)",
 	)
 }
+
+// This makes sure that the priority class we create is
+// present and lower than the system priority classes. Example:
+// openshift-user-critical   1000000000   false            66m
+// system-cluster-critical   2000000000   false            114m
+// system-node-critical      2000001000   false            114m
+func TestToEnsureUserPriorityClassIsPresentAndLower(t *testing.T) {
```

Comment:
@s-urbaniak wdyt of this test case, is this something like what you had in mind? Was not sure where to place it, as it does relate to user workload monitoring but since the class is created before it can be part of the main as well? Suggestions welcome!

Reply:
suggestion: maybe let's place it in a separate `test/e2e/priority_class_test.go`?

Reply:
Sounds good as well!